index.html

```diff
@@ -4,8 +4,23 @@
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="./css/styles.css" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="./css/styles.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Meine Seite</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1 class="ich">Efendi Varol</h1>
+      <nav class="nav">
+        <a href="#">Menu</a>
+        <a href="#">Projektanfragen</a>
+        <a href="#">Kontakt</a>
+        <a href="#">Skills</a>
```

Comment:
I'm very sure you wanted to do this later anyways: "#Menu", "#Projektanfragen" etc. to distinguish between the links